src/tenants/salesforce/VTOCampaign2025/styles/ImageSlideshow.module.scss

```
@@ -0,0 +1,77 @@
+  @include lgLaptopView {
+    font-size: $fontXXLarge;
+    line-height: 100%;
+  }
+}
+
+.swiperWrapper {
+  max-width: 960px;
+  margin: 0 auto;
+  overflow: hidden;
+
+  :global(.swiper) {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)